plugin/pom.xml

```diff
@@ -191,17 +191,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>io.jenkins.plugins</groupId>
-            <artifactId>theme-manager</artifactId>
```

Comment:
only needed as a JavaScript API now so not needed to be in pom. We could leave it in bumping the version to ensure that the plugins work together but I don't think needed